### PR TITLE
doc: updated readme of v1.0

### PR DESCRIPTION
**Pull request by oldLady344**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,22 +1,41 @@
-# nebula-java
+# nebula-java v1.0
 
 ![](https://img.shields.io/badge/language-java-orange.svg)
 [![star this repo](http://githubbadges.com/star.svg?user=vesoft-inc&repo=nebula-java&style=default)](https://github.com/vesoft-inc/nebula-java)
 [![fork this repo](http://githubbadges.com/fork.svg?user=vesoft-inc&repo=nebula-java&style=default)](https://github.com/vesoft-inc/nebula-java/fork)
 
-This guide provides instructions and options for connecting **Nebula Graph** for Java developer. However, Nebula Java is not thread-safe.
+Nebula Java is a Java client for Java developers to connect their projects to Nebula Graph.
```
Comment (Nicole00) [inline README.md:7]: Nebula java is not just a client, but also includes exchange 1.0.1/ spark-connector 1.0.1/ flink-connector 1.0.1

File: README.md [hunk 2]
```diff
@@ -1,22 +1,41 @@
...
+
+> **NOTE**: Nebula Java is not thread-safe.
+
+This branch, v1.0, is for Nebula Java v1.0. This table lists the versions of Nebula Java that can work with Nebula Graph v1.1.0 and earlier versions.
```
Comment (Nicole00) [inline README.md:11]: no need to mention branch v1.0

File: README.md [hunk 3]
```diff
@@ -26,11 +45,11 @@ When using Maven, add dependency to your `pom.xml` file:
...
 ```
 
-For more versions, please refer to [releases](https://github.com/vesoft-inc/nebula-java/releases).
+If Nebula Java v1.0.1 is used, use `<version>1.0.1</version>` in the dependency.
 
-### Graph Client Example
+### Graph client example
 
-Connect to the `graphd`:
+Connect to the `nebula-graphd` process:
```
Comment (Nicole00) [inline README.md:52]: nebula-graphd service?

File: README.md [hunk 4]
```diff
@@ -51,37 +70,39 @@ Execute a query:
...
-Q: Error occurs when building from the source code.
+Q: An error occurs during the process of building from the source code.
 
 ```text
 Failed to execute goal org.apache.maven.plugins:maven-gpg-plugin:1.6:sign (default) on project client: Exit code: 2 -> [Help 1]
 org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal org.apache.maven.plugins:maven-gpg-plugin:1.6:sign (default) on project client: Exit code: 2
 ```
 
-A: This means that you need to have a key to sign the jars
+A: You must generate a key to sign the jars
```
Comment (Nicole00) [inline README.md:105]: or user can skip the gpg:
`mvn clean compile package install -Dgpg.skip -Dmaven.javadoc.skip=true -Dtest.skip=true -Dmaven.test.skip=true`

**Review by laura-ding — APPROVED** [review]

**Pull request merged**